sapi/fuzzing/fuzzer-exif.c

```diff
@@ -0,0 +1,74 @@
+/*
+   +----------------------------------------------------------------------+
+   | PHP Version 7                                                        |
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Stanislav Malyshev <stas@php.net>                           |
+   +----------------------------------------------------------------------+
+ */
+
+#include "fuzzer.h"
+
+#include "Zend/zend.h"
+#include "main/php_config.h"
+#include "main/php_main.h"
+#include "ext/standard/php_var.h"
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "fuzzer-sapi.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+	char *filename;
+	int filedes;
+
+	if (php_request_startup()==FAILURE) {
+		php_module_shutdown();
+		return 0;
+	}
+
+	/* put the data in a file */
+	filename = tmpnam(NULL);
+	filedes = open(filename, O_CREAT|O_RDWR);
+	write(filedes, Data, Size);
+	close(filedes);
+
+	fuzzer_call_php_func("exif_read_data", 1, &filename);
+
+	/* cleanup */
+	unlink(filename);
+	php_request_shutdown(NULL);
+
+	return 0;
+}
+
+int LLVMFuzzerInitialize(int *argc, char ***argv) {
+	fuzzer_init_php();
+
+	/* fuzzer_shutdown_php(); */
+	return 0;
+}
+
+/*
```

Comment:
Local variables can be maybe also removed as was done recently in favor of the EditorConfig. 92ac598aabd336593a47ed3959f1031674b763e6